app/_layout.tsx

```diff
@@ -15,6 +15,7 @@ const Layout = () => {
           <Stack.Screen name="login" options={{ headerShown: false }} />
         </Stack>
       </ThemeWrapper>
+      <PrivyElements/>
```

Comment:
This is a requirement for privy related stuff right?
What is the difference between this and `PrivyProvider`?

Reply:
`PrivyProvider` is the wrapper that provides the application with use of all privy hooks. 
`PrivyElements` serves privy UI elements, which we currently do not use but may want to explore using as crutches. I am removing it for now.